packages/edit-site/src/components/save-panel/index.js

```diff
@@ -130,7 +139,8 @@ export default function SavePanel() {
 						variant="secondary"
 						className="edit-site-editor__toggle-save-panel-button"
 						onClick={ () => setIsSaveViewOpened( true ) }
-						aria-expanded={ false }
+						aria-haspopup={ 'dialog' }
+						ref={ saveBtnRef }
```

Comment:
I think the bug here is that the save panel button gets _removed_ when `isSaveViewOpen` is `true`. Because the save panel button is removed, `<_EntitiesSavedStates onClose={ onClose } />` loses its reference for where to return focus to onClose. If we restructure this section so that the button isn't removed but hidden when `isSaveViewOpen === true`, the focus should return correctly. I'll do a quick PR to demo. Might be easier to explain that way.